RevenueCatUI/PaywallView.swift

```diff
@@ -268,6 +277,9 @@ struct LoadedOfferingPaywallView: View {
     @Environment(\.locale)
     private var locale
 
+    @Environment(\.onRequestedDismissal)
+    private var onRequestedDismissal: (() -> Void)?
```

Comment:
Hmm maybe I'm wrong but I think we could make this not nullable now? (We always pass a value I think)

Reply:
there will only be a value if `onRequestedDismissal` on PaywallView is called, and it's not even a public function (but PaywallView is), so this needs to be nullable

Reply:
Ohh ok makes sense